README.md

```diff
@@ -23,14 +23,14 @@ Input Example with `OK Button Caption`(Optional):
    input message: 'input-message', ok: 'OK'
 
 Input Example with `Allowed Submitter`:
-User IDs and/or external group names of person or people permitted to respond to the input, separated by ','. Spaces will be trimmed automatically, so "Alice, Bob, Charles" is the same as "Alice,Bob,Charles".
+User ids and/or external group names of person or people permitted to respond to the input, separated by ','. Spaces will be trimmed automatically, so "Alice, Bob, Charles" is the same as "Alice,Bob,Charles".
 Note: Jenkins administrators are able to respond to the input regardless of the value of this parameter.
 
    input message: '', submitter: 'jenkins-user, jenkins-user2'
 
 Input Example with Parameter to store the `Approving Submitter`:
 
-If specified, this is the name of the return value that will contain the ID of the user that approves this input. The return value will be handled in a fashion similar to the parameters value.
+If specified, this is the name of the return value that will contain the id of the user that approves this input. The return value will be handled in a fashion similar to the parameters value.
```

Comment:
Username rather than `id` (I believe, though I'm not thrilled with having two occurrences of `user` in the same sentence.
```suggestion
If specified, this is the name of the return value that will contain the username of the user that approves this input. The return value will be handled in a fashion similar to the parameters value.
```

Reply:
done